### DISCUSSION TITLE
Can you show NERF in a web browser ?

### DISCUSSION THREAD

**Discussion by cyz2727327** [NVlabs/instant-ngp#881] [Q&A] [answered]
Hey guys, does anyone know if it's possible to show this NERF 3D rendering in a web browser, that is compatible and integrable with the web technology.

I am only asking because on this instantNerf webpage it seems possible.

Any ideas would be welcome !

**Comment by Gyudori** [comment, marked as answer]
If you mean [https://nvlabs.github.io/instant-ngp/](url) this project page, I think it's only displaying mp4 videos for demonstration.

On the other hand, there is a project that you asked. It's called SNeRG. 
With this method, you can bake nerf and visualize it in web pages.
Here's a link. [https://phog.github.io/snerg/](url)

**Reply by cyz2727327** [reply]
thank you this looks promising, I will take a closer look

**Reply by IGNITEgm** [reply]
> If you mean [https://nvlabs.github.io/instant-ngp/](url) this project page, I think it's only displaying mp4 videos for demonstration.
> 
> On the other hand, there is a project that you asked. It's called SNeRG. With this method, you can bake nerf and visualize it in web pages. Here's a link. [https://phog.github.io/snerg/](url)

The link died.  I am also interested in whether it is possible to pass it to a 3D mesh format (OBJ, FBX...) with the texture information or directly to another direct format for the web (GLTF, GLB...) and if it is possible, how to do it?
